roles/ansible_service_broker/tasks/install.yml

```diff
@@ -312,6 +312,9 @@
                 org:  {{ ansible_service_broker_registry_organization }}
                 tag:  {{ ansible_service_broker_registry_tag }}
                 white_list: {{ ansible_service_broker_registry_whitelist }}
+              - type: local_registry
```

Comment:
What is the purpose of this?

Reply:
These were changes I pulled in from #6026 to fix problems associated with actually enabling ASB by default. That change has merged and I've rebased that commit out.